specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+          },
+          {
+            "$ref": "#/parameters/ignoreRelationships"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AtlasEntityWithExtInfo"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "Update entity partially - Create/Update entity attribute identified by its GUID.",
+        "description": "Update entity partially - create or update entity attribute identified by its GUID.\nSupports only primitive attribute type and entity references.\nIt does not support updating complex types like arrays, and maps.\nNull updates are not possible.",
+        "operationId": "Entity_PartialUpdateEntityAttrByGuid",
+        "x-ms-examples": {
+          "EntityREST_PartialUpdateEntityAttrByGuid": {
+            "$ref": "./examples/EntityREST_PartialUpdateEntityAttrByGuid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "description": "The name of the attribute.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The value of the attribute.",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated with EntityMutationResponse.",
+            "schema": {
+              "$ref": "#/definitions/EntityMutationResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "Delete an entity identified by its GUID.",
+        "description": "Delete an entity identified by its GUID.",
+        "operationId": "Entity_DeleteByGuid",
+        "x-ms-examples": {
+          "EntityREST_DeleteByGuid": {
+            "$ref": "./examples/EntityREST_DeleteByGuid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/guid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted with EntityMutationResponse.",
+            "schema": {
+              "$ref": "#/definitions/EntityMutationResponse"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/entity/guid/{guid}/classification/{classificationName}": {
+      "get": {
+        "tags": [
+          "EntityREST"
+        ],
+        "summary": "List classifications for a given entity represented by a GUID.",
+        "description": "List classifications for a given entity represented by a GUID.",
+        "operationId": "Entity_GetClassification",
```

Comment:
Would this be better as something like `EntityClassification_Get`?  Not saying you have to, but at present there are 34 methods on `Entity` which seems a bit fat.  If there's a more logical way to group this it might help make the API surface area easier to reason about.

Comment also applies to `Glossary`

Reply:
What we want is to perform get classification operation on entity class, not get operation on classification class. This is designed to follow Atlas behavior.